scripts/README.md

```diff
@@ -0,0 +1,98 @@
+# scripts
+
+This directory contains javascripts file to build and prepare packages.
+
+## buildPackages.js
+
+```javascript
+'use strict';
+
+const fs = require('fs');
```

Comment:
Do we really need the entire code in README?

Reply:
This is just 1 line readme otherwise.

Reply:
I have seen some readme files which contains the entire code of important scripts files.
![Screenshot (16)](https://user-images.githubusercontent.com/43860289/77232349-8195db80-6bc6-11ea-883d-79af8cdc7e5c.png)
https://github.com/webpack/webpack/tree/master/examples/aggressive-merging

therefore I inserted the whole code in the readme.

Reply:
Can do that for important files like config as your example, but not sure if everyone needs to read the buildPackages (an example) code in a README, if they want they can always refer the actual file fwiw.

Reply:
np i will remove these

Reply:
![Screenshot (17)](https://user-images.githubusercontent.com/43860289/77232767-f0743400-6bc8-11ea-9988-ed660a1acbb3.png)
@anshumanv however these are also some of the files which are displayed and i think that they are not as important as the config file

Reply:
We have a separate documentation site altogether with additional docs, for devs who want to actually get started with the codebase, what we have right now should suffice IMO. Anyways, that's just my say, would wait for thoughts from others. 😄

Reply:
I agree with @anshumanv